README.md

```diff
@@ -34,6 +34,18 @@ version can be found on in the maven repository for
   <artifactId>simpleclient_pushgateway</artifactId>
   <version>0.0.20</version>
 </dependency>
+<!-- Guava Cache metrics-->
```

Comment:
Don't expand this list, it's already annoying to maintain and I don't want to scare off users.